source/index.js

```diff
@@ -53,7 +53,7 @@ module.exports = async (input = 'patch', options) => {
 		options.cleanup = false;
 	}
 
-	const pkg = util.readPkg();
+	const pkg = util.readPkg(options.contents);
```

Comment:
There is no default value for `--contents`, and so you should use the CWD here as a default value, to make sure that `util.readPkg` is called with a valid path.

Reply:
I looked into that and read-pkg-up defaults it to process.cwd when called with undefined.

Would you still like me to change it to be more explicit?
`const pkg = util.readPkg(options.contents || process.cwd());`

Reply:
I think it would be better to update `readPkg` to use `process.cwd()` as a default value instead.

Reply:
(It's better to use `rootDir` as that's the directory the package is located in, and it might be different from `process.cwd()`.)

Reply:
Also, shouldn't you use an absolute path instead of a relative path here? `readPkg` can still accept relative paths, but we'll eventually need to pass `read-pkg-up` an absolute path.